src/allotropy/parsers/appbio_quantstudio_designandanalysis/appbio_quantstudio_designandanalysis_calculated_documents.py

```diff
@@ -216,22 +216,23 @@ def build_delta_ct_mean(
     else:
         return None
 
-    if r_adj_eq_ct_mean_ref := build_adj_eq_ct_mean(view_data, sample, r_target):
-        data_sources.append(
-            DataSource(
-                feature="adjusted equivalent ct mean",
-                reference=r_adj_eq_ct_mean_ref,
+    if r_target is not None:
```

Comment:
Nit: do 

```
if r_target is None:
    return None

...
```

This will let you unindent the larger body

Reply:
the objective of this function is basically to construct the data_source list

the first value is added before this piece of code in lines 203 or 210

the second value is added or not depending on r_target. 
But if r_target is none I still have the first result. 

If there is a value in r_target I have to call build_adj_eq_ct_mean or build_eq_ct_mean trying to construct the related result, only when both functions fail I have to return the None value of line 235.